src/components/FilterItem.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Paper from '@mui/material/Paper';
+
+const FilterItem = styled(Paper)(({ theme }) => ({
+  backgroundColor: '#C6C6C6',
+  ...theme.typography.body2,
+  borderRadius: "15px",
+  padding: theme.spacing(0.5),
+  textAlign: 'center',
+  color: 'white',
+}));
```

Comment:
Might look a bit better with a white background, black text and a shadow that ressembles the one used on figma, you may wnat to experiement on figma then see whant looks best, or not if you dont have the time or dont care...